awx/main/conf.py

```diff
@@ -345,6 +346,61 @@ def _load_default_license_from_file():
     category_slug='jobs',
 )
 
+register(
+    'AWX_RESOURCE_PROFILING_ENABLED',
+    field_class=fields.BooleanField,
+    default=False,
+    label=_('Enable resource profiling on all tower jobs'),
+    help_text=_('If set, resource profiling data will be collected on all jobs.'),  # noqa
```

Comment:
Going back to some of the design discussions - we want these to be named and help texted in such a way that:
- it describes what it's for (sending data to Red Hat)
- it will not conflict with what we may do in the future
-

Reply:
@jladdjr will you file a ticket to follow up on this?

Reply:
yep :+1

Reply:
created stub here: https://github.com/ansible/awx/issues/4923

filling this out now